frontend/src/Pages/StudentManagement.tsx

```diff
@@ -171,70 +173,79 @@ export default function StudentManagement() {
                         />
                     </div>
 
-                    <div
-                        className="tooltip tooltip-left"
-                        data-tip="Add Student"
-                    >
+                    <div className="tooltip tooltip-left" data-tip="Add User">
                         <button
-                            className="btn btn-primary btn-sm"
+                            className="btn btn-primary btn-sm text-white"
                             onClick={() => addUserModal.current?.showModal()}
                         >
-                            <UserPlusIcon className="h-4" />
+                            <PlusIcon className="h-4 border border-white rounded-full p-1 mr-2" />{' '}
+                            Add Student
```

Comment:
You won't need to add these classes to this icon, and rather than it being imported from`outline`, it should be imported from`solid` (the other icons on this page under "Actions" should be imported from `outline`, though!).

Reply:
Totally my bad - you were right on this of it being outline!